lexi_meter/game_logic/quiz.py

```diff
@@ -0,0 +1,3 @@
+class Quiz:
```

Comment:
it's a good practice when creating a File to use a doc string to give a description of what the file will hold `"""The Quiz file will hold all logic when accessing Quiz data."""` for an example.

Reply:
Thank you Matthew, changes made 👍